http.go

```diff
@@ -322,7 +329,87 @@ func httpPutViolation(w http.ResponseWriter, r *http.Request) {
 	// Force object field and type to match value specified in request path
 	v.Object = valstr
 	v.Type = typestr
-	httpPutViolationsInner(w, r, typestr, []ViolationRequest{v})
+	if err := putViolation(typestr, v); err != nil {
+		w.WriteHeader(http.StatusBadRequest)
```

Comment:
Should we be capturing the error in the log here since what was `httpPutViolationsInner` became `putViolation` and we do not log errors there now?